File: Sakila_Net/CS_SakilaLib/Neg/NegFilm.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegFilm
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOFilm.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClFilm film) 
        {
            return DAO.DAOFilm.sqlSelect(film);
        }
        public  static int crear(ClFilm film)
        {
            return DAO.DAOFilm.sqlInsert(film);
        }
        public  static int eliminar(ClFilm film)
        {
            return DAO.DAOFilm.sqlDelete(film);
        }
        public  static int actualizar(ClFilm film)
        {
            return DAO.DAOFilm.sqlUpdate(film);
        }
    }
}

File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegFilm.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegFilm
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOFilm.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( film as ClFilm) as Boolean
                        Return DAO.DAOFilm.SelectOne(film)     
                    End Function
                    Public  Shared Function crear( film as ClFilm) as Integer
                        Return DAO.DAOFilm.SqlInsert(film)     
                    End Function
                    Public  Shared Function eliminar( film as ClFilm) as Integer
                        Return DAO.DAOFilm.SqlDelete(film)     
                    End Function
                    Public  Shared Function actualizar( film as ClFilm) as Integer
                        Return DAO.DAOFilm.SqlUpdate(film)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
